app/src/main/java/com/hacka/presenteperfeito/feature/home/screen/util/GetFirstName.kt

```diff
@@ -0,0 +1,5 @@
+package com.hacka.presenteperfeito.feature.home.screen.util
+
+fun getFirstName(fullName: String): String {
```

Comment:
Maninho poderia ficar no pacote Core>Common>Extensions, pois pode ser que seja utilizado essa logica fora do contexto da home.
```suggestion
fun String.getFirstName(): String {
    return this.split(" ").firstOrNull() ?: ""
}
```